$NetBSD: patch-configure,v 1.1 2019/11/18 18:34:09 ng0 Exp $

portability fix, from upstream commit 7f49525e811bee4c88ec5ca1d959ec1f9e2590cf.

--- configure~  2019-11-18 18:13:11.117643620 +0000
+++ configure   2019-11-18 18:14:04.396905402 +0000
@@ -42844,7 +42844,7 @@
     ESNI_ENABLED=0
   ESNI_SUPPORT=''
 
-      if test "x$OPENSSL_ENABLED" == "x1"; then
+      if test "x$OPENSSL_ENABLED" = "x1"; then
     for ac_func in SSL_get_esni_status
 do :
   ac_fn_c_check_func "$LINENO" "SSL_get_esni_status" "ac_cv_func_SSL_get_esni_status"
@@ -42860,7 +42860,7 @@
 
     fi
 
-    if test "x$ESNI_ENABLED" == "x1"; then
+    if test "x$ESNI_ENABLED" = "x1"; then
 
 $as_echo "#define USE_ESNI 1" >>confdefs.h
